<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="exit_action"/>
    <addaction name="connect_action"/>
    <addaction name="disconnect_action"/>
    <addaction name="clear_action"/>
    <addaction name="settings_action"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="connect_action"/>
   <addaction name="disconnect_action"/>
   <addaction name="clear_action"/>
   <addaction name="settings_action"/>
   <addaction name="exit_action"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exit_action">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/closingsession_theoutputnear_cierredesesion_10028.png (exit)</normaloff>:/actions/images/closingsession_theoutputnear_cierredesesion_10028.png (exit)</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="connect_action">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/usbcable_78354.png (disconnect)</normaloff>:/actions/images/usbcable_78354.png (disconnect)</iconset>
   </property>
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
  <action name="disconnect_action">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/connect_1217.png (connect)</normaloff>:/actions/images/connect_1217.png (connect)</iconset>
   </property>
   <property name="text">
    <string>Отключиться</string>
   </property>
  </action>
  <action name="clear_action">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/actions_edit_clear_15784.png (clear)</normaloff>:/actions/images/actions_edit_clear_15784.png (clear)</iconset>
   </property>
   <property name="text">
    <string>Очистить консоль</string>
   </property>
  </action>
  <action name="settings_action">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/Settings_30027.png (settings)</normaloff>:/actions/images/Settings_30027.png (settings)</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
